data_model/__init__.py

```diff
@@ -140,4 +144,40 @@ class Features(Base):
     parent = Column( ForeignKey("cdm.features.id"), comment="Parent feature for this feature if nested")
 
 
+class Users(Base):
+    __tablename__ = "users"
+    __table_args__ = {'schema': 'cdm'}
+    id = Column(  String, comment="ID / primary key", primary_key=True)
+    username = Column(String, comment="User name")
 
+
+# StationsRoleMixin = resource_role_class(
```

Comment:
@chinedu117 is it still possible for additional tables introduced by `oso` to be created by `resource_role_class`?